### ISSUE TITLE
Custom color save/load/paint problem

### ISSUE THREAD

**Issue by magrelow** [reaper-oss/sws#1108]
Hi,
![screen shot 2019-02-08 at 15 38 22](https://user-images.githubusercontent.com/47501121/52535623-c39fed00-2d37-11e9-9460-4a1ea8329e65.png)
![screen shot 2019-02-08 at 15 39 24](https://user-images.githubusercontent.com/47501121/52535624-c39fed00-2d37-11e9-9c48-6d3ff9ec09cc.png)

 I'm having a problem to set the custom colors. All 20 boxes are filled but when I set my tracks to "set to ordered customs color" reaper paints the first 10 correctly, paint 6 black and start from the 1st color. Any clue? 
Also, I can save and load the color preset but it doesn't work, saving and loading doesn't change the color boxes.

Mac osx 10.12.6
Reaper 5.965 64
SWS 2.10 and tried 2.9.8 as well, same problem

**Comment by nofishonfriday** [comment]
Hi in SWS there are 16 different custom colors as seen in your first screenshot (not 20).
From your screenshot it looks like SWS custom colors 11-16 are set to black, so when running "set to ordered customs color" it seems to me it's working correctly, setting tracks 1 - 16 to the current SWS custom colors (with tracks 11- 16 black), then repeating.

**Comment by magrelow** [comment]
I see what you mean. The thing is if SWS picks the first 16 colors out of the 20 boxes (and they are all painted) there shouldn't be black painted tracks like I have on track 11 to 16. I guess these boxes are the ones I have to paint or there is another window to set up the custom colors?

**Comment by nofishonfriday** [comment]
> I guess these boxes are the ones I have to paint
Yes, I think you have to set them initially, if that's what you mean.

I can only test on Windows currently, here it works like this (long screencapture, sorry):
![cust col](https://user-images.githubusercontent.com/7658194/52803445-60d58b00-3082-11e9-8fb3-41ab9d55b9e9.gif)

Does this work for you on Mac too?

**Comment by magrelow** [comment]
No, it's completly different on Mac. But hey, I found the problem. If I stretch the window to the right it shows the correct sequence of colors. Thanks anyway buddy!
![screen shot 2019-02-14 at 15 05 44](https://user-images.githubusercontent.com/47501121/52804036-5c9d7380-306a-11e9-803b-dc342280ee56.png)

**Comment by nofishonfriday** [comment]
Oh nice, you're welcome. :D 
So can we close this or should it stay open because of the save / load issue?

**Comment by magrelow** [comment]
The save and load isn't really important for me but I just tested and it still not working.
